consensus/hotstuff/committees/cluster_committee.go

```diff
@@ -101,10 +102,10 @@ func (c *Cluster) Identity(blockID flow.Identifier, nodeID flow.Identifier) (*fl
 	// otherwise use the snapshot given by the reference block
 	identity, err := c.state.AtBlockID(payload.ReferenceBlockID).Identity(nodeID)
 	if err != nil {
-		return nil, fmt.Errorf("could not get identity for node (id=%x): %w", nodeID, err)
+		return nil, fmt.Errorf("could not get identity for node (id=%x): %w", nodeID, convertError(err))
```

Comment:
With error handling, we are still struggling with failed attempts of uniform error handling patterns. I feel we should air on the opposite side for now and treat errors on a case-by case basis (no functions for uniform error handling. I am just very worried that at some point, somebody will decide to change the `convertError` method and just log the error and drop it.)

Hence, I feel we could just do the error conversion explicitly here.